user_tools/src/spark_rapids_pytools/cloud_api/emr.py

```diff
@@ -114,9 +114,6 @@ def create_saving_estimator(self,
     def create_local_submission_job(self, job_prop, ctxt) -> Any:
         return EmrLocalRapidsJob(prop_container=job_prop, exec_ctxt=ctxt)
 
-    def _get_prediction_model_name(self) -> str:
```

Comment:
Previously we did not have a pretrained model for emr, we used to default to `onprem`.